src/garage/tf/policies/base.py

```diff
@@ -23,41 +23,68 @@ def __init__(self, name, env_spec):
         self._cached_param_shapes = None
 
     @abc.abstractmethod
-    def get_action(self, observation):
+    def get_action(self, observation, agent_infos):
```

Comment:
i think this should be a kwarg with default `None`. recurrent policies can raise a `ValueError` if it's not specified, but there's no reason to force everyone to pass something for it.

Reply:
fixed